### PR TITLE
bump to 2.8.0

### PR DESCRIPTION
**Pull request by akelad**

### CODE REVIEW

File: .github/workflows/continuous_deployment.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ env:
   # * rasa version in requirements.txt
   # * rasa-sdk base image version in Dockerfile
   RASA_X_VERSION: "0.40.1"
```
Comment (melindaloubser1) [inline .github/workflows/continuous_deployment.yml:36]: We'll need to wait till we can bump this before merging since 2.8.0 is too high for 0.40.1

Comment (melindaloubser1) [inline .github/workflows/continuous_deployment.yml:36]:
```suggestion
  RASA_X_VERSION: "0.42.0"
```

**Review by melindaloubser1 — APPROVED** [review]

File: .github/workflows/continuous_deployment.yml [hunk 2]
```diff
@@ -33,8 +33,8 @@ env:
   # NB! The rasa/rasa-x versions should be in sync with:
   # * rasa version in requirements.txt
   # * rasa-sdk base image version in Dockerfile
-  RASA_X_VERSION: "0.40.1"
-  RASA_VERSION: "2.6.3"
+  RASA_X_VERSION: "0.42.0"
+  RASA_VERSION: "2.8.0"
```
Comment (melindaloubser1) [inline .github/workflows/continuous_deployment.yml:37]:
```suggestion
  RASA_VERSION: "2.8.2"
```

File: Dockerfile [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # * rasa version in requirements.txt
 # * RASA_VERSION and RASA_X_VERSION  in .github/workflows/continuous-deployment.yml
 # Pull SDK image as base image
-FROM rasa/rasa-sdk:2.6.0
+FROM rasa/rasa-sdk:2.8.0
```
Comment (melindaloubser1) [inline Dockerfile:5]:
```suggestion
FROM rasa/rasa-sdk:2.8.2
```

File: requirements.txt [hunk 4]
```diff
@@ -1,4 +1,4 @@
-rasa==2.6.2 # NB! when updating, make sure to also update:
+rasa==2.8.0 # NB! when updating, make sure to also update:
```
Comment (melindaloubser1) [inline requirements.txt:1]:
```suggestion
rasa==2.8.2 # NB! when updating, make sure to also update:
```

**Review by melindaloubser1 — APPROVED** [review]

**Comment by akelad** [comment]
@melindaloubser1 should this be merged?

**Comment by melindaloubser1** [comment]
Yes, it should, but it never did because the discourse API calls in the unit tests are failing

we can force merge it and open an issue to look into that separately, since it means the existing version's calls also won't work anyway.

**Comment by akelad** [comment]
I would rather we fix it, if we have the time 😅

**Comment by melindaloubser1** [comment]
Ok, in that case I can get to it later this week

**Pull request merged**